ENIGMAsystem/SHELL/Platforms/Win32/WINDOWScallback.cpp

```diff
@@ -200,11 +200,13 @@ namespace enigma
         keybdstatus[key]=1;
         if (key!=18)
         {
-          if ((lParam&(1<<29))>0)
-               last_keybdstatus[18]=keybdstatus[18], keybdstatus[18]=1;
-          else last_keybdstatus[18]=keybdstatus[18], keybdstatus[18]=0;
+          if ((lParam&(1<<29))>0) {
```

Comment:
dafux is 29 & 18? plz use a const bro

Reply:
It's old code I didn't write it, and I just got done answering that to myself, it's the ALT key. I was going to ask Josh if I can use the actual UPPERCASE virtual key constants there like I did for my new code since they do not conflict with the enigma user keycodes. Also I have no idea why the person was checking key instead of wParam directly.